README.rst

```diff
@@ -165,16 +187,29 @@ Writing a health check is quick and easy:
     from health_check.backends import BaseHealthCheckBackend
 
     class MyHealthCheckBackend(BaseHealthCheckBackend):
+        def __init__(self):
+            super().__init__()
+
+            # Display name on the endpoint.
+            self.identifier = 'My HealthCheck'
+
+            # A useful description of the health check's purpose, which
+            # can be when the status is returned.
+            self.description = 'An awesome and useful health check!'
```

Comment:
I would suggest to make this a class not instance attribute.
https://www.python-course.eu/python3_class_and_instance_attributes.php